packages/a11y/README.md

```diff
@@ -39,15 +39,15 @@ speak( 'The message you want to send to the ARIA live region', 'assertive' );
 
 _Parameters_
 
--   _message_ `string`: The message to be announced by Assistive Technologies.
--   _ariaLive_ `string`: Optional. The politeness level for aria-live. Possible values: polite or assertive. Default polite.
+-   _message_ `string`: The message to be announced by assistive technologies.
+-   _ariaLive_ `[(|)]`: The politeness level for aria-live; default: 'polite'.
```

Comment:
There is one issue with this PR and it's right here: the auto-generated docs aren't filling in the type info correctly. I think it should read `[('assertive'|'polite')]`. I could override it, but then next time the docs were auto-generated, it would get overriden. Any ideas?